src/reporter.ts

```diff
@@ -81,31 +62,21 @@ class StandardReporter implements Reporter {
     testResult: TestResult,
     results: AggregatedResult
   ): void {
-    this.status.testFinished(test.context.config, testResult, results)
     this.output.testFinished(test.path, testResult, results)
 
     if (!testResult.skipped) {
-      this.stdlib.log(
-        getResultHeader(testResult, this.globalConfig, test.context.config)
-      )
-
       if (
         this.globalConfig.verbose &&
         !testResult.testExecError &&
         !testResult.skipped
       ) {
-        this.stdlib.log(getTestResults(testResult.testResults))
+        // no-op
       }
 
       if (testResult.failureMessage) {
-        this.stdlib.error(testResult.failureMessage)
+        // no-op
```

Comment:
There seems to be an if statement in line 67 now that contains two more if statements and none of those 3 statements do anything. Can they all be removed?

Reply:
Yeah. I think they can go now.